humble/distribution.yaml

```diff
@@ -6328,6 +6328,12 @@ repositories:
       url: https://github.com/vrpn/vrpn.git
       version: master
     status: maintained
+  wall_follower_ros2:
+    source:
+      type: git
+      url: https://github.com/rfzeg/wall_follower_ros2.git
+      version: master
```

Comment:
I think this should be main, `master` doesn't exist on your source repo

Reply:
Sorry about that, fixed.